ui/src/app/services/error-handling.service.ts

```diff
@@ -67,20 +68,16 @@ export class ErrorHandlingService {
           this.setServerBusy();
           throw e;
         }
-        switch (e.status) {
+
+        this.errorResponse = this.convertAPIError(e);
```

Comment:
I don't like binding a class level variable at the method level, because if multiple errors are hitting at once, I worry about churn and a race condition being established here. Can we make this a method scoped variable instead?